### PR TITLE
[Merged by Bors] - Add more paths to HTTP API metrics

### PR DESCRIPTION
**Pull request by michaelsproul**
## Proposed Changes

Expand the set of paths tracked by the HTTP API metrics to include all paths hit by the validator client.

These paths were only partially updated for Altair, so we were missing some of the sync committee and v2 APIs.

### CODE REVIEW

File: beacon_node/http_api/src/lib.rs [hunk 1]
```diff
@@ -200,18 +200,25 @@ pub fn prometheus_metrics() -> warp::filters::log::Log<impl Fn(warp::filters::lo
...
-                .or_else(|| starts_with("v1/validator/blocks"))
-                .or_else(|| starts_with("v2/validator/blocks"))
                 .or_else(|| starts_with("v1/validator/aggregate_attestation"))
                 .or_else(|| starts_with("v1/validator/aggregate_and_proofs"))
+                .or_else(|| starts_with("v1/validator/sync_committee_contribution"))
+                .or_else(|| starts_with("v1/validator/contribution_and_proofs"))
                 .or_else(|| starts_with("v1/validator/beacon_committee_subscriptions"))
+                .or_else(|| starts_with("v1/beacon/pool/attestations"))
+                .or_else(|| starts_with("v1/beacon/pool/sync_committees"))
+                .or_else(|| starts_with("v1/beacon/blocks/head/root"))
```
Comment (michaelsproul) [inline beacon_node/http_api/src/lib.rs:218]: This query is used by the VC to get the block root for sync committee contributions

**Review by paulhauner — APPROVED** [review]
LGTM!

**Comment by michaelsproul** [comment]
I'll add the sync committee subscriptions endpoint before merging

**Pull request closed without merging**